Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM ubuntu:focal-20221019
+FROM ubuntu:focal-20221130
 
-ARG VERSION=15.6.2
+ARG VERSION=15.7.0
 
 ENV GITLAB_VERSION=${VERSION} \
     RUBY_VERSION=3.0.4 \
     RUBY_SOURCE_SHA256SUM="70b47c207af04bce9acea262308fb42893d3e244f39a4abc586920a1c723722b" \
```

Comment:
I guess we can update to ruby 3.0.5 (see <https://gitlab.com/gitlab-org/gitlab-omnibus-builder/-/blob/master/docker/VERSIONS>).

Please change to
```
RUBY_VERSION=3.0.4 \
RUBY_SOURCE_SHA256SUM="9afc6380a027a4fe1ae1a3e2eccb6b497b9c5ac0631c12ca56f9b7beb4848776"
```
(see <https://www.ruby-lang.org/de/downloads/>).